internal/controller/nodeop_controller.go

```diff
@@ -920,33 +911,44 @@ func (r *NodeOpReconciler) createRebootPod(ctx context.Context, nodeOp *kairosio
 					Command: []string{
 						"/bin/sh",
 						"-c",
-						fmt.Sprintf(`
-							echo "=== Checking for existing reboot annotation ==="
-							EXISTING_ANNOTATION=$(kubectl get pod $POD_NAME --namespace $POD_NAMESPACE -o jsonpath='{.metadata.annotations.kairos\.io/reboot-state}' 2>/dev/null || echo "")
-							if [ "$EXISTING_ANNOTATION" = "%s" ]; then
-								echo "Reboot annotation already exists, reboot was already performed. Exiting successfully."
-								exit 0
-							fi
-							echo "No reboot annotation found, proceeding with reboot process..."
-							ls -la /sentinel/ || echo "Cannot list /sentinel directory"
-							while true; do
-								SENTINEL_FILE=$(find /sentinel -name "%s-*" -type f 2>/dev/null | head -1)
-								if [ -n "$SENTINEL_FILE" ]; then
-									echo "Found sentinel file: $SENTINEL_FILE"
-									echo "Deleting sentinel file before reboot..."
-									rm -f "$SENTINEL_FILE"
-									echo "Attempting to patch pod..."
-									kubectl patch pod $POD_NAME -p '{"metadata":{"annotations":{"kairos.io/reboot-state":"%s"}}}' --namespace $POD_NAMESPACE || echo "kubectl patch failed"
-									echo "Giving 5 seconds to the Job Pod to exit gracefully..."
-									sleep 5
-									echo "Attempting reboot with nsenter..."
-									nsenter -i -m -t 1 -- reboot || echo "nsenter reboot failed"
-									break
-								fi
-								echo "No matching sentinel file found, sleeping..."
-								sleep 10
-							done
-						`, rebootStatusCompleted, jobBaseName, rebootStatusCompleted),
+						`echo "=== Checking for existing reboot annotation ==="
```

Comment:
The changes here are:
- interpolate the variables using `+` to make it clear which one goes where
- Cleanup any leftover matching sentinels before we enter the `while` loop